src/app/pages/dynamic-stepper/dynamic-stepper.component.scss

```diff
@@ -0,0 +1,37 @@
+@import '~@pxblue/colors/palette.scss';
+
+.radio-group {
+    display: flex;
+    flex-direction: column;
+}
+
+.delete_icon {
+    vertical-align: middle;
+    color: map-get($pxb-black, 500);
+    font-size: 18px;
+    line-height: 24px;
+}
+
+.add_button {
+    width: 100%;
+    height: 100%;
+    padding: 16px 64px;
+    background: transparent;
+    text-align: left;
+
+    &[disabled] {
+        background: transparent;
+    }
+}
+
+::ng-deep .mat-step:last-child .mat-step-label {
```

Comment:
do we need ng deep on these (e.g., if we change the viewEncapsulation)?

Reply:
looks like we do need ng-deep here regardless of ViewEncapsulation, unless I can find a more specific selector. Haven't had any luck yet.